server_installation/topics/operator/external-database.adoc

```diff
@@ -0,0 +1,137 @@
+
+[[_external_database]]
+=== Connecting to an external database
+
+You can use the Operator to connect to an external PostgreSQL database by modifying the Keycloak custom resource and creating a `keycloak-db-secret` YAML file. Note that values are Base64 encoded.
+
+.Example YAML file for `keycloak-db-secret`
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: keycloak-db-secret
+    namespace: keycloak
+stringData:
+    POSTGRES_DATABASE: <Database Name>
+    POSTGRES_EXTERNAL_ADDRESS: <External Database IP or URL (resolvable by K8s)>
+    POSTGRES_EXTERNAL_PORT: <External Database Port>
+    # Strongly recommended to use <'Keycloak CR Name'-postgresql>
+    POSTGRES_HOST: <Database Service Name>
+    POSTGRES_PASSWORD: <Database Password>
+    # Required for AWS Backup functionality
+    POSTGRES_SUPERUSER: true
+    POSTGRES_USERNAME: <Database Username>
+ type: Opaque
+```
+
+The following properties set the hostname or IP address and port of the database.
+
+* `POSTGRES_EXTERNAL_ADDRESS` - an IP address or a hostname of the external database.
+ifeval::[{project_community}==true]
+This address needs be resolvable in a Kubernetes cluster.
+endif::[]
+* `POSTGRES_EXTERNAL_PORT` - (Optional) A database port.
+
+The other properties work in the same way for a hosted or external database. Set them as follows:
+
+* `POSTGRES_DATABASE` - Database name to be used.
+* `POSTGRES_HOST` - The name of the `Service` used to communicate with a database. Typically `keycloak-postgresql`.
+* `POSTGRES_USERNAME` - Database username
+* `POSTGRES_PASSWORD` - Database password
+* `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
+
+The Keycloak custom resource requires updates to enable external database support. 
+
+.Example YAML file for `Keycloak` custom resource that supports an external database
+[source,bash,subs=+attributes]
```

Comment:
On the line 8 there is used different type of formatting ```yaml```. Can we use the same type of code formatting also here?

Reply:
@andymunro I do not know is you observed this comment. So I am just assigning it to you. Thank you for your help. :-)